tests/common.go

```diff
@@ -7183,22 +7183,24 @@ func EndPxBackupTorpedoTest(contexts []*scheduler.Context) {
 	if ginkgoTestDescr.Failed {
 		log.Infof(">>>> FAILED TEST: %s", ginkgoTestDescr.FullTestText)
 	}
-	log.Infof(">>>> Collecting logs for testcase : %s", ginkgoTestDescr.FullTestText)
-	testCaseName := ginkgoTestDescr.FullTestText
-	matches := regexp.MustCompile(`\{([^}]+)\}`).FindStringSubmatch(ginkgoTestDescr.FullTestText)
-	if len(matches) > 1 {
-		testCaseName = matches[1]
-	}
-	masterNode := node.GetMasterNodes()[0]
-	log.Infof("Creating a directory [%s] to store logs", pxbLogDirPath)
-	err := runCmd(fmt.Sprintf("mkdir -p %v", pxbLogDirPath), masterNode)
-	if err != nil {
-		log.Errorf("Error in creating a directory [%s] to store logs. Err: %v", pxbLogDirPath, err.Error())
-		return
+	if Inst().Provisioner != "ibm" || Inst().Provisioner != "gke" {
```

Comment:
can you use GetClusterProviders()[0] == “gke" for both gke and ibm .. since provisioner might change in case of multiple provisioner support

Reply:
If we do that then it'll run in case of gke which is not what we want.

Reply:
It was just an example i tried to quote use GetClusterProviders()  instead of Inst().Provisioner
we  need to use GetClusterProviders() which in turn uses CLUSTER_PROVIDER , as provisioner might change in case of multiple provisioner support

Reply:
will will discuss with mithun and merge it after shutdown

Reply:
I have changed to logic which will work in all cases.